init.lua

```diff
@@ -315,15 +326,19 @@ minetest.register_chatcommand("getrank", {
 	params = "<name> | name of player",
 	func = function(name, param)
 		if param and param ~= "" then
-			if minetest.get_player_by_name(param) then
-				local rank = ranks.get_rank(param) or "No rank"
+			if storage:get_string(param) then
+				local rank = storage:get_string(param):gsub("^%l", string.upper)
 				return true, "Rank of "..param..": "..rank
 			else
-				return false, "Invalid player \""..name.."\""
+				return false, "Rank of "..param..": No rank"
 			end
 		else
-			local rank = ranks.get_rank(name) or "No rank"
-			return false, "Your rank: "..rank
+			if storage:get_string(name) then
+				local rank = storage:get_string(name):gsub("^%l", string.upper) or "No rank"
+				return false, "Your rank: "..rank
+			else
+				return false, "Your rank: No rank"
+			end
```

Comment:
Does any of this really need to change thanks to `ranks.get_rank`?

Reply:
I'm working on this now! I've found a better way to do it, with less code.

Reply:
I've changed it.